server/monitoring/monitoring.go

```diff
@@ -48,6 +51,7 @@ func Monit() {
 		} else {
 			if found > 0 {
 				logger.Info("monitoring", fmt.Sprintf("Deleted %d entries and files in %s", found, time.Since(now)))
+				statistics.S.UpdateCurrentSize(sizeRemoved)
```

Comment:
I dont' think it matters? Just tested and got same results.
Anyway, the stats could probably be moved inside database.DB.Update{} where a transaction is already started to make monitoring run a bit faster (tried this and it does run a bit faster).